### DISCUSSION TITLE
Using go plugins with Netdata from Ubuntu repository

### DISCUSSION THREAD

**Discussion by deadnews** [netdata/netdata#13010] [Q&A] [answered]
https://github.com/netdata/go.d.plugin/blob/c7d7fc23f2515965cd9a5d329221cf87bdc8d377/README.md?plain=1#L22
>Go.d.plugin is shipped with [Netdata](https://github.com/netdata/netdata).
- Netdata installed with [kickstart.sh](https://github.com/netdata/netdata/blob/master/packaging/installer/methods/kickstart.md):
```bash
❯ find /opt/netdata -name *go.d*
/opt/netdata/usr/lib/netdata/conf.d/go.d.conf
/opt/netdata/usr/lib/netdata/conf.d/go.d
/opt/netdata/usr/lib/netdata/conf.d/health.d/go.d.plugin.conf
/opt/netdata/usr/libexec/netdata/plugins.d/go.d.plugin
/opt/netdata/etc/netdata/go.d
```

- Ubuntu (22.04) [package](https://packages.ubuntu.com/jammy/netdata-core):
```bash
❯ curl -s https://packages.ubuntu.com/jammy/amd64/netdata-core/filelist | grep go.d
/usr/lib/netdata/conf.d/health.d/go.d.plugin.conf
```
Ubuntu [has packages](https://packages.ubuntu.com/search?searchon=names&suite=jammy&keywords=netdata): `netdata-plugins-bash`, `netdata-plugins-nodejs`, `netdata-plugins-python`. 
But there is no `netdata-plugins-go` package.

So, it's not possible to use [go plugins](https://github.com/netdata/go.d.plugin#available-modules) with Netdata from the Ubuntu repository?

**Comment by ilyam8** [comment]
Hi, @DeadNews. The kickstart script uses [our repository](https://packagecloud.io/netdata/netdata) when installing netdata. The packages in the Ubuntu repo are not maintained by us.

**Comment by deadnews** [comment, marked as answer]
References:
https://answers.launchpad.net/ubuntu/+source/netdata/+question/701962
https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1009754
